reviews/graduation-coredns.md

```diff
@@ -0,0 +1,51 @@
+
+# CoreDNS Graduation Application
+
+CoreDNS joined CNCF in March 2017 as inception project and graduated as incubation project on February 2018
+
+It has grown significantly over time either as a DNS Server available in Cloud Native environment, either as a key component of Kubernetes project:
+- In Fall 2017, CoreDNS was proposed to become a DNS Discovery Service for Kubernetes.
+- First version as alpha feature was released in v1.9 of Kubernetes - Dec 2017
+- CoreDNS is considered the “default” DNS Discovery Service for Kubernetes v1.13 - to be released in Dec 2018
+
+The following application links to the required information to become a graduated project.
+
+## CoreDNS fulfills all the incubating and graduation criteria:
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+
+* Bose, SoundCloud, Infoblox, Zalando, Tradeshift, Yandex, AdGuard are knows users of CoreDNS. More adopters are listed [here](https://github.com/coredns/coredns/blob/master/ADOPTERS.md).
+
+* On Kubernetes side, from [a recent CNCF Survey](https://docs.google.com/forms/d/e/1FAIpQLSehsvd0n2HKnPEifukNHej4gMBrXe2saJOmxAmPq34dTGMWKA/viewanalytics), 50% deployments of cluster are using CoreDNS as the DNS discovery service
+
+
+### Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+
+CoreDNS currently has 16 maintainers.
+They are listed as ‘approvers’ on our [OWNER document](https://github.com/coredns/coredns/blob/master/OWNERS)
+
+### Demonstrate a substantial ongoing flow of commits and merged contributions.
+
+More than [100+ contributors](https://github.com/coredns/coredns/graphs/contributors) and 12 [releases](https://coredns.devstats.cncf.io/d/3/community-stats?orgId=1&var-period=d&var-repo_name=CoreDNS) since incubation graduation
+
+### Have committers from at least two organizations.
+
+We currently have [16 maintainers](https://github.com/coredns/coredns/blob/master/OWNERS).
+Most of them are contributing on their spare time and each considered at its own organization,
```

Comment:
Most of them are contributing in their spare time; but, considering organizations, 4 maintainers are working for 2 distinct companies

Reply:
ok